pkg/controller/statefulset/stateful_set.go

```diff
@@ -98,21 +100,26 @@ func NewStatefulSetController(
 			history.NewHistory(kubeClient, revInformer.Lister()),
 			recorder,
 		),
-		pvcListerSynced: pvcInformer.Informer().HasSynced,
-		revListerSynced: revInformer.Informer().HasSynced,
-		queue:           workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "statefulset"),
-		podControl:      controller.RealPodControl{KubeClient: kubeClient, Recorder: recorder},
-
+		pvcListerSynced:  pvcInformer.Informer().HasSynced,
+		revListerSynced:  revInformer.Informer().HasSynced,
+		queue:            workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "statefulset"),
+		podControl:       controller.RealPodControl{KubeClient: kubeClient, Recorder: recorder},
```

Comment:
This looks like an unrelated change that just happened because one blank line got removed, forcing a reformatting.

You don't need to if this is the only thing that needs to be changed, but it would be cleaner to remove this change.

Reply:
This is still open.